Resources/Locale/en-US/guidebook/chemistry/effects.ftl

```diff
@@ -267,6 +267,12 @@ reagent-effect-guidebook-make-sentient =
         *[other] make
     } the metabolizer sentient
 
+reagent-effect-guidebook-make-polymorph =
+    { $chance ->
+        [1] Polymorphs
+        *[other] polymorph
+    } the metabolizer into a { $protoname }
```

Comment:
not gonna push on it but there might be some way to automatically do the `a`/`an` replacement? Might be hard since you're passing in the name, which is a string.

Reply:
I think there should some sort of system for it instead of a one-off thing for this effect. If suppose just checking if the first letter of the string starts with an `a, e, i, o,` or `u` - then use `an`, otherwise - a.